### PR TITLE
allow player rename

### PR DESCRIPTION
**Pull request by anthonyng43**
just as title suggests... i'm not good with describing stuff

### CODE REVIEW

**Comment by anthonyng43** [comment]
added car tune handling and ghost level handling

File: src/modules/cars/functions.ts [hunk 1]
```diff
@@ -724,52 +819,90 @@ export async function checkCreatedCar(body: wm.protobuf.CreateCarRequest, car: a
...
     return { cheated }
+}
+
+
+// Check Create Car Name
+export async function checkNameInput(body: any) {
+    let slurName: boolean = false;
+
+    // List of slur names to check against
+    let allSlurName = [
```
Comment (lostkagamine) [inline src/modules/cars/functions.ts:878]: this is the wrong way to do this, i don't think moderating this automatically is worth it though

**Review by lostkagamine — COMMENTED** [review]
raising an eye at the ghost name automod stuff but other than that lgtm

File: src/modules/cars/functions.ts (hunk 1 above)
Comment (anthonyng43) [reply src/modules/cars/functions.ts:878]: i see, i'll remove automod stuff while keep custom gt wing fix and some ocm changes

**Comment by anthonyng43** [comment]
should be ok now i think

**Pull request merged**